apps/web/src/components/QualifiedRecruitmentCard/utils.ts

```diff
@@ -37,7 +37,11 @@ const getAvailabilityInfo = (
     };
   }
 
-  if (isPlacedStatus(status?.value)) {
+  // placed casual is an exception, it can be suspended
+  if (
```

Comment:
Because this logic is so confusing already, it might help if we name an intermediate variable.  Something like:
```ts
const isLongTermPlacedStatus =  isPlacedStatus(status?.value) &&
    status?.value !== PoolCandidateStatus.PlacedCasual;
```

Reply:
Sure https://github.com/GCTC-NTGC/gc-digital-talent/pull/12741/commits/9c65393a70a701de5761e915625d8e761c2826ea